taxonomy/management/commands/refresh_program_skills.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""
+Management command for refreshing the skills associated with programs.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.utils.translation import gettext as _
+
+from taxonomy import utils
+from taxonomy.exceptions import ProgramMetadataNotFoundError, InvalidCommandOptionsError
+from taxonomy.models import RefreshProgramSkillsConfig
+from taxonomy.providers.utils import get_program_metadata_provider
+
+LOGGER = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+        Command to refresh skills associated with the programs.
+
+        Example usage:
+            $ ./manage.py refresh_program_skills --program 'program1_uuid' --program 'program2_uuid' --commit
+            $ # args-from-database means command line arguments will be picked from the database.
+            $ ./manage.py refresh_program_skills --args-from-database
+            $ # To update all the programs
+            $ ./manage.py refresh_program_skills --all --commit
```

Comment:
Our underlying assumption here is that Programs have been published/active, correct?

Reply:
yes! it is